olm.sh

```diff
@@ -24,8 +24,10 @@ function install_binaries() {
 
 install_binaries
 
+RELEASE="$(sed -n 's/^.*DefaultOperatorImage = "minio\/operator:v\(.*\)"/\1/p' pkg/controller/operator.go)"
+
 # get the minio version
-minioVersionInExample=$(kustomize build examples/kustomization/tenant-openshift | yq eval-all '.spec.image' | tail -1)
```

Comment:
I checked `examples/kustomization/tenant-openshift`, but it looks almost like `examples/kustomization/tenant-lite`, but without the `containerSecurityContext`. I guess we could remove this folder completely, because it seems to be unused now.

Reply:
oh no, we might not be using it in here but we want to keep it, this is used on the samples and I did a copy of tenant-lite instead of referencing it on purpose.

We are using tenant-openshift here:

https://github.com/minio/operator/blob/01c56a5a17303641aa9b5dd4017428d585125c11/samples/kustomization.yaml#L1-L3

And that sample is used on the Openshift layovers:

https://github.com/minio/operator/blob/01c56a5a17303641aa9b5dd4017428d585125c11/config/manifests/kustomization.yaml#L3-L7
To finally be rendered as the example Tenant in the clusterserviceversion (CSV):
https://github.com/minio/operator/blob/01c56a5a17303641aa9b5dd4017428d585125c11/bundles/certified-operators/5.0.14/manifests/minio-operator.clusterserviceversion.yaml#L5-L80

The final goal is to pre-render an example Tenant yaml that will be shown in the Openshift console when you click the "Create Tenant" button.

![Screenshot from 2024-04-22 10-20-53](https://github.com/minio/operator/assets/1334362/a29c9cb0-5c4d-4833-92ca-f566bf7edb68)